src/SearchIndex/index.ts

```diff
@@ -48,9 +48,13 @@ export class SearchIndex {
     const metaAggregationQuery = query.getMetaQuery(searchProperty, this.convertedTaxonomy);
     const cursor = this.documents.aggregate(metaAggregationQuery);
     try {
-      const aggRes = await cursor.toArray();
-      const res = formatFacetMetaResponse(aggRes[0] as FacetAggRes, this.convertedTaxonomy);
-      return res;
+      // TODO: re-implement
```

Comment:
Currently, the `/v2/search/meta` route will only shallow search the base level of facets and accept facet names to drill down.

The follow up PR will update this to be *expand all facets and accept filters for all levels of facets*. Return this in the format outlined in DOP-4048 for easy front end consumption

This is not required for the first pass at Search Facets UI (as they will be based off the `/v2/status/` route)